extensions/mssql/src/hdfs/fileStatus.ts

```diff
@@ -3,12 +3,34 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-export enum HdfsFileType {
+import { FileType } from '../objectExplorerNodeProvider/fileSources';
+
+export const enum HdfsFileType {
 	File = 'File',
 	Directory = 'Directory',
 	Symlink = 'Symlink'
 }
 
+/**
+ * Maps a @see HdfsFileType to its corresponding @see FileType. Will return undefined if
+ * passed in type is undefined.
+ * @param hdfsFileType The HdfsFileType to map from
+ */
+export function hdfsFileTypeToFileType(hdfsFileType: HdfsFileType | undefined) {
```

Comment:
Can you add `: FileType` as the return type definition here?

Reply:
Gah, thanks for catching this 😄